integration-tests/src/main/java/org/apache/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -114,6 +116,21 @@ private void loadProperties(String configFile)
         noClientAuthRouterTLSUrl = StringUtils.format("https://%s:%s", noClientAuthRouterHost, props.get("router_no_client_auth_tls_port"));
       }
     }
+    customCertCheckRouterUrl = props.get("router_no_client_auth_url");
+    if (customCertCheckRouterUrl == null) {
+      String customCertCheckRouterHost = props.get("router_no_client_auth_host");
+      if (null != customCertCheckRouterHost) {
```

Comment:
nit: i find this kind of jarring since outer is `var == null` and then inside it's flipped to `null != var`, but I don't think it's a big deal

Reply:
hm, it follows the convention for the other nodes in that file, i'll leave this unchanged in this PR